msvc-full-features/vcpkg.json

```diff
@@ -3,11 +3,19 @@
     "version-string": "0.F",
     "dependencies": [
         "sdl2",
-        "sdl2-image",
+        {
+          "name": "sdl2-image",
+          "features": [ "libjpeg-turbo" ]
+        },
         {
           "name": "sdl2-mixer",
           "features": [ "libflac", "mpg123", "libmodplug", "libvorbis" ]
         },
-        "sdl2-ttf"
+        "sdl2-ttf",
+        {
+          "features": [ "tools" ],
+          "name": "gettext",
```

Comment:
I agree, in fact in my original PR text I wrote:
> Build VCPKG's whole gettext just to use msgfmt. Since it take too long, maybe will be removed in favor of using the one in _Git for Windows_.
I had a doubt since even if required by this project's instruction. _Git for Windows_ is not really needed by Visual Studio because it uses its own. So, I guess I will go for plan B and use `gettext.exe` from _Git for Windows_ and remove `gettext` from `vcpkg.json` completely.

Reply:
How does FIND_PACKAGE work exactly? Like what if I had my own msys2 install and environment, could I run cmake/vscode from there and have it pick up msgfmt from that distro and avoid a git for windows dependency entirely?

Reply:
It now uses `find_program` with paths to msys2 first and git for windows after.